app/code/Magento/Backend/Block/Store/Switcher.php

```diff
@@ -113,8 +114,7 @@ public function __construct(
     protected function _construct()
     {
         parent::_construct();
-
-        $this->setUseConfirm(true);
```

Comment:
maybe we can add a check if not set - then set it to true?

Reply:
Maybe i should add check like this 

`if (!$this->hasData('use_confirm')) {
            $this->setUseConfirm(true);
        } elseif ($this->hasData('use_confirm')) {
            $this->setUserConfirm((bool)$this->getData('use_confirm'));
        }`

and remove public additional method

Reply:
Maybe like this?
```php
$this->setUserConfirm($this->hasData('use_confirm') ? (bool)$this->getData('use_confirm') : true);
```

Reply:
Ok i have updated that